src/plugins/monthSelect/index.ts

```diff
@@ -133,19 +123,22 @@ function monthSelectPlugin(pluginConfig?: Partial<Config>): Plugin {
 
     function selectYear() {
       let selectedDate = fp.selectedDates[0];
-      if (selectedDate) {
-        selectedDate = new Date(selectedDate);
-        selectedDate.setFullYear(fp.currentYear);
-        if (fp.config.minDate && selectedDate < fp.config.minDate) {
-          selectedDate = fp.config.minDate;
-        }
-        if (fp.config.maxDate && selectedDate > fp.config.maxDate) {
-          selectedDate = fp.config.maxDate;
-        }
-        fp.currentYear = selectedDate.getFullYear();
-        fp.currentYearElement.value = String(fp.currentYear);
-        fp.currentMonth = selectedDate.getMonth();
+      
+      selectedDate = selectedDate 
```

Comment:
Can you elaborate a bit about why you made this change?

Is it only because "the code looks better" (refactoring) or does it change how the picker actually works?

Reply:
@jensgustafsson This PR should have been closed: as mentioned in my comment above, this same bug was fixed in #2278.

@MolnApps, just realized I never said thanks for your contribution, and sorry to clobber your PR with my own, later one. Wish I had seen it first!

Reply:
I see. That means we haven't updated the docs to run the latest version yet?

Reply:
Can you clarify what you mean? [This is only page in the docs](https://flatpickr.js.org/plugins/#monthselectplugin) that mentions the monthSelect plugin, and nothing here needs to be updated to reflect this bugfix (AFAICT).

In general, none of the bugfixes I've done so far have come with any changes to the documentation.